.github/workflows/tf-gke-cluster.yaml

```diff
@@ -0,0 +1,55 @@
+name: Terraform Create GKE Cluster
+# Reusable workflow to create a GKE cluster using Terraform
+
+on:
+  workflow_call:
+    inputs:
+      tf-plan:
+        description: "Name of the terraform plan output file"
+        required: false
+        default: "tfplan"
+        type: string
+      working-directory:
+        description: "Directory containing your Terraform project. (Default is .)"
+        required: false
+        default: "."
+        type: string
+    secrets:
+      google-credentials:
+        description: "Key of the service account to use."
+        required: true
+
+jobs:
+  terraform:
+    name: "Terraform preprocessing"
```

Comment:
Why `preprocessing`?

Reply:
that was because i was thinking of removing the `plan` and `apply` it and let `atlantis` manage it. But since it's not currently used, i can change the name i guess